dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java

```diff
@@ -603,7 +614,7 @@ private boolean doesOrganisationExistToUser(Long organisationId, Long userId) {
 
         // Check that you are not applying action on yourself
         if (Objects.equals(user.getId(), userId)) {
-            String msg = "You cannot add yourself to an organisation.";
+            String msg = "You cannot modify yourself in an organisation.";
```

Comment:
Thought for discussion, should this be true. 
What happens when curators/admins add themselves to organizations currently?

Reply:
Currently they are able to add themselves.

Reply:
I may be misunderstanding then. 
Is a different check operating for curators/admins?

Reply:
Oh sorry, no one should be able to edit their own organisation membership. At least not yet, if we decide to change that.